packages/tags/README.md

```diff
@@ -83,3 +89,9 @@ import {
     </sp-tag>
 </sp-tags>
 ```
+
+### Accessibility
+
+`<sp-tags>` is a `role="list"` container that manages a collection of `Tag` elements. It has an `aria-label` attribute that defaults to "Tags".
+
+`<sp-tags>` uses the roving tabindex pattern for efficient keyboard navigation. `Tab` enters the collection, arrow keys navigate between tags, and only deletable tags are focusable. The container provides `role="list"` semantics with each tag as a `role="listitem"` for proper screen reader support.
```

Comment:
oh oh oh! I was just looking into roving tabindex. Want to link to the documentation? 

`/tools/roving-tab-index`